assets/src/components/form/inputs/base/input-label.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External imports
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * generates an html <label> tag
+ *
+ * @function
+ * @param {string} label
+ * @param {string} htmlFor
+ * @param {string} htmlClass
+ * @param {boolean} required
+ * @param {boolean} strong
+ * @param {Object} children
+ * @param {Object} attributes
+ * @return {string} rendered form label
+ */
+export const InputLabel = ( {
```

Comment:
eeeyaa... I don't see how that would be of much use here as I would likely have to always just use the `singular` value from the `Label` VO and form builders would have to generate a VO simply to pass a single string...

there might be a use case where using a VO would work, but I can't think of one off the top of my head... and again... I can't see how we would decide which prop from the VO to use...

hmmmm... (thinking as I'm typing)... it could possibly work for cases where you have a dynamic number of options for a set of checkbox or radio button inputs and you want to change the main label for the group... but even in that scenario, it might be better to do that detection in the form itself, decide on the appropriate label text, and then simply pass that text to this component.

Did you have a specific scenario in mind where using the `Label` VO would work?